lib/local-links-manager/import/enabled_service_checker.rb

```diff
@@ -15,8 +15,16 @@ def initialize(csv_downloader = CsvDownloader.new(CSV_URL))
 
       def enable_services
         @csv_downloader.each_row do |row|
```

Comment:
Personal preference so feel free to ignore: Could we use the one-liner versions (ie. `@csv_downloader.each_row { |row| supported_lgsl_codes.add(row["LGSL"]) }`) instead?